drape_frontend/frontend_renderer.cpp

```diff
@@ -414,17 +412,18 @@ void FrontendRenderer::OnResize(ScreenBase const & screen)
   m_contextFactory->getDrawContext()->resize(viewportRect.SizeX(), viewportRect.SizeY());
   RefreshProjection();
 
-  if (m_useFramebuffer)
+  if (screen.isPerspective())
   {
     int width = screen.GetWidth();
     int height = screen.GetHeight();
     int const maxSide = max(width, height);
-    if (maxSide > m_framebuffer->GetMaxSize())
+    int const maxTextureSize = m_framebuffer->GetMaxSize();
+    if (maxSide > maxTextureSize)
     {
-      width = width * m_framebuffer->GetMaxSize() / maxSide;
-      height = height * m_framebuffer->GetMaxSize() / maxSide;
-      LOG(LINFO, ("Max texture size:", m_framebuffer->GetMaxSize(), ", expanded screen size:", maxSide,
-                  ", scale:", m_framebuffer->GetMaxSize() / (double)maxSide));
+      width = width * maxTextureSize / maxSide;
```

Comment:
Please, declare a constant for maxTextureSize / maxSide and remove that code duplication.